RunTests.ps1

```diff
@@ -0,0 +1,77 @@
+
+$DnxRuntimes = @(
+	'dnx-clr-win-x86.1.0.0-beta4', 
+	'dnx-coreclr-win-x86.1.0.0-beta4');
+
+$TestProjects = @(
+	'.\test\Microsoft.ApplicationInsights.AspNet.Tests',
+	'.\test\Mvc6Framework45.FunctionalTests',
+	'.\test\WebApiShimFw46.FunctionalTests'
+)
+
+Function Execute-Process {
```

Comment:
Why do we need custom code for executing `dnx test` command and parsing its output?